content-src/components/Base/Base.jsx

```diff
@@ -106,6 +120,25 @@ export class BaseContent extends React.PureComponent {
   constructor(props) {
     super(props);
     this.openPreferences = this.openPreferences.bind(this);
+    this.onWindowScroll = debounce(this.onWindowScroll.bind(this), 5);
```

Comment:
I made the wait only 5ms because even then when you scroll fast you can see some jank when the header appears. Better than nothing?

Reply:
(I guess we can tweak it based on feedback once it lands?)